cost-analyzer/values.yaml

```diff
@@ -249,6 +249,34 @@ global:
       enabled: false  # Set to true when using affected CI/CD tools for access to the below configuration options.
       skipSanityChecks: false  # If true, skip all sanity/existence checks for resources like Secrets.
 
+  ## Kubecost Integrations (beta)
+  ## Ref: https://docs.kubecost.com/integrations
+  ##
+  integrations:
+    postgres:
+      enabled: false
+      runInterval: "1h"  # How frequently to run the integration.
+      databaseHost: ""  # REQUIRED. ex: my.postgres.database.azure.com
+      databasePort: ""  # REQUIRED. ex: 5432
+      databaseName: ""  # REQUIRED. ex: postgres
+      databaseUser: ""  # REQUIRED. ex: myusername
+      databasePassword: ""  # REQUIRED. ex: mypassword
+
+      ## Configure the parameters when querying Kubecost's Allocation API. And
+      ## configure what database table to write the data to. Refer to the API
+      ## docs for further details:
+      ## https://docs.kubecost.com/apis/apis-overview/api-allocation
+      allocations:
+        enabled: true
+        queryConfigs:
+          - databaseTable: "kubecost_allocation_data"
+            window: "7d"
+            aggregate: "namespace"
+            idle: "true"
+            shareIdle: "false"
+            shareNamespaces: ""
+            shareLabels: ""
```

Comment:
Main question here is what is the behavior inside cost-model if the inputs provided here are wrong, for example:

1. the value of a param is of the wrong type
2. a param is omitted
3. the API changes to include a new field (required) and it is omitted (see 2)
4. a key in a param here is wrong (ex., "windw" instead of "window")

Just making sure that we're somehow account for these, logging something helpful and, above all, not crashing/panicing. I realize this isn't exactly a question for this Helm change, but it's obviously related.

Reply:
As an update, the Postgres Integration now runs inside the Aggregator container. It runs as a separate goroutine so if it were to crash all other aggregator processes would carry on as usual!

If (1) (2) or (3) occurs, the API request to the aggregator would fail and the job would exit with an error message of the failure.

If (4) occurs, it would be similar to omitting a parameter and and the job would exit if the API request fails without that parameter.